tests/api/EventFunnelCest.php

```diff
@@ -90,25 +100,36 @@ public function showStatsByTemporaryUserId(ApiTester $I)
                 'skip_call_booking' => 10,
                 'thankyou_page' => 10,
                 'leave' => 10,
-                'sign up' => 10
+                'sign up' => 10,
+                'summarization' => [
```

Comment:
You should compare the expected behavior of statistics with the real behavior, real response to detecting issues in your code, and test yourself.